File: sample_code/read_from_file.py
import csv

def problem(line):
	# TODO: Fill this out

def read_file():
	with open('input.csv') as input_file:
		reader = csv.reader(input_file, delimiter=',')
		lines = []

		for i, row in enumerate(reader):
			if i > 0:
				lines.append(row)
		return lines
	
if __name__ == '__main__':
	input_lines = read_file()

	for line in input_lines:
		problem(line)

File: sample_code/ReadFromFile.java
import java.util.Scanner;
import java.io.File;
import java.io.FileNotFoundException;

public class ReadFromFile {
	public static void main(String[] args) {
		File file = new File("input.csv");
		try {
			Scanner s = new Scanner(file);

			int n = Integer.parseInt(s.next());
			while(s.hasNext()) {
				String line = s.next();
				problem(line);
			}
			s.close();
		} catch (FileNotFoundException e) {
			e.printStackTrace();
		}
	}

	public static void problem(String line) {
		//TODO: Fill this out
	}

	public static String[] separate(String stringToSeparate) {
		return stringToSeparate.split(",");
	}
}
